vendor/README.md

```diff
@@ -612,7 +612,6 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Visual Basic .NET:** [peters-ben-0007/VBDotNetSyntax](https://github.com/peters-ben-0007/VBDotNetSyntax)
 - **Visual Basic 6.0:** [serkonda7/vscode-vba](https://github.com/serkonda7/vscode-vba)
 - **Volt:** [textmate/d.tmbundle](https://github.com/textmate/d.tmbundle)
-- **Vue:** [vuejs/vue-syntax-highlight](https://github.com/vuejs/vue-syntax-highlight)
```

Comment:
We need the correct version of this line.

Reply:
Done in https://github.com/github-linguist/linguist/pull/7086/commits/63f916e41b8ace8ebc44603d6136e0a16ebeb940